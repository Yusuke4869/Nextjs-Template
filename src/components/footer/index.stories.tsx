import { Meta, StoryObj } from "@storybook/react";

import Footer from ".";

type T = typeof Footer;
type Story = StoryObj<T>;

const data = {};

export default {
  component: Footer,
  args: { ...data },
  argTypes: {},
} as Meta<T>;

export const Default: Story = {};
