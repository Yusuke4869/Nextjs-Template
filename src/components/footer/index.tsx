import type { FC } from "react";

const Footer: FC = () => (
  <footer>
    <p>Footer</p>
  </footer>
);

export default Footer;
