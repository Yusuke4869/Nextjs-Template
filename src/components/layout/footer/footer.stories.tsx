import { ComponentMeta, ComponentStoryObj } from "@storybook/react";

import Footer from ".";

type T = typeof Footer;
type Story = ComponentStoryObj<T>;

const data = {};

export default {
  component: Footer,
  args: { ...data },
  argTypes: {},
} as ComponentMeta<T>;

export const Default: Story = {};
