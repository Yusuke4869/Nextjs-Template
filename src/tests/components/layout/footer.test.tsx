import { composeStories } from "@storybook/testing-react";
import { render } from "@testing-library/react";
import * as stories from "~/components/layout/footer/footer.stories";

const { Default } = composeStories(stories);

describe("(components) footer", () => {
  test("Snap Shot", () => {
    const { container } = render(<Default />);
    expect(container).toMatchSnapshot();
  });
});
