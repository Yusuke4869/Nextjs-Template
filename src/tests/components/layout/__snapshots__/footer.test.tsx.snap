// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(components) footer Snap Shot 1`] = `
<div>
  <footer>
    <p>
      Footer
    </p>
  </footer>
</div>
`;
